sim/paladin/protection/presets.go

```diff
@@ -60,115 +70,31 @@ var FullIndividualBuffs = &proto.IndividualBuffs{
 }
 
 var FullConsumes = &proto.Consumes{
-	Flask:           proto.Flask_FlaskOfBlindingLight,
-	Food:            proto.Food_FoodBlackenedBasilisk,
-	DefaultPotion:   proto.Potions_SuperManaPotion,
+	Flask:           proto.Flask_FlaskOfStoneblood,
+	Food:            proto.Food_FoodDragonfinFilet,
+	DefaultPotion:   proto.Potions_IndestructiblePotion,
+	PrepopPotion:	 proto.Potions_IndestructiblePotion,
 	DefaultConjured: proto.Conjured_ConjuredDarkRune,
 }
 
 var FullDebuffs = &proto.Debuffs{
-	Misery:          true,
-	CurseOfElements: true,
-	BloodFrenzy:     true,
-	SunderArmor:     true,
-	FaerieFire:      proto.TristateEffect_TristateEffectImproved,
-	CurseOfWeakness: proto.TristateEffect_TristateEffectImproved,
+    JudgementOfWisdom: true,
+    JudgementOfLight: true,
+    Misery: true,
+    FaerieFire: proto.TristateEffect_TristateEffectImproved,
+    EbonPlaguebringer: true,
+    TotemOfWrath: true,
+    ShadowMastery: true,
+    BloodFrenzy: true,
+    Mangle: true,
+    ExposeArmor: true,
+    SunderArmor: true,
+    Vindication: true,
+    ThunderClap: proto.TristateEffect_TristateEffectImproved,
+    InsectSwarm: true,
 }
 
 var P1Gear = items.EquipmentSpecFromJsonString(`{"items": [
 ]}`)
 var Phase4Gear = items.EquipmentSpecFromJsonString(`{"items": [
-	{
```

Comment:
Deleting this gear makes sense since it's from TBC, but shouldn't it be replaced with a new set?

Removing this preset and only using the P1Gear would be fine too. In case you haven't figured it out yet, everything in this file is used *only* for the unit tests.

Reply:
I definitely had not figured that out... Thank you.